### PR TITLE
Update PIO dependencies

### PR DESCRIPTION
**Pull request by TimoWielink**
**Describe at a high level the solution you're providing**
Add crypto and CRC dependencies for PIO

**Is this a patch, a minor version change, or a major version change**
Patch 2.1.16

### CODE REVIEW

File: examples/1.Ducks/DetectorDuck/platformio.ini [hunk 1]
```diff
@@ -19,8 +19,9 @@ monitor_speed = 115200
 monitor_filters = time
 
 lib_deps = 
-    ClusterDuck Protocol
-    ; /PATH-TO/ClusterDuck-Protocol
+    /Users/timowielink/Documents/Arduino/libraries/ClusterDuck-Protocol
+    /Users/timowielink/Documents/Arduino/libraries/ClusterDuck-Protocol/libraries
+    https://github.com/Project-Owl/Crypto.git
```
Comment (krook) [inline examples/1.Ducks/DetectorDuck/platformio.ini:25]: Looks like you have some workstation-specific lines here @TimoWielink

Comment (TimoWielink) [reply examples/1.Ducks/DetectorDuck/platformio.ini:25]: Let me change that!

**Comment by krook** [comment]
Is this ready to merge?

**Comment by krook** [comment]
Good to merge @TimoWielink?

**Pull request merged**